src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
-body {
+@import './utils/variables';
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+  width: calc(
+    #{$size-of-cell} * #{$number-of-columns} + #{$gap} * 6 + #{$padding} * 2
+  );
+
+  &__day {
+    height: 100px;
+    width: 100px;
+    border: 1px solid #000;
+    background-color: $color-of-cell;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    transition: all $duration-of-animation;
```

Comment:
Avoid using 'all' in the 'transition' property. It's more performant to specify which properties you are transitioning, for example, 'transition: background-color $duration-of-animation, transform $duration-of-animation'.